### DISCUSSION TITLE
Kvantum's svg rendering order confusing

### DISCUSSION THREAD

**Discussion by vaxerski** [tsujan/Kvantum#964] [Q&A]
Hi, couldn't find anything in the making-themes pdf, but what is the rule for rendering the elements in kvantum?

I am trying to make a border around Menus, and in Inkscape, and Firefox, I get:
![20240322_16h39m25s](https://github.com/tsujan/Kvantum/assets/43317083/16ed42e4-ae1f-4b1b-8bc0-08de7834a228)

These are basically 2 shapes stacked on top of each other, one is scaled to be bigger and behind the original one, with the color of the border:

<details>
  <summary>svg</summary>
  
```svg
<g
     style="fill-rule:nonzero"
     id="menu-shadow-left"
     transform="translate(135.25741,-386.99685)">
    <path
       style="opacity:1;fill:#84362e;fill-opacity:1;stroke-width:1.02942"
       d="M -109.29849,1091.7501 H -104 v 48 h -5.29849 z"
       id="path22" />
    <path
       style="opacity:1;fill:#110E0E"
       d="m -109,1091.7501 h 5 v 48 h -5 z"
       id="path3" />
    <path
       style="opacity:.35;fill:url(#linearGradient6752)"
       d="m -116.2495,1091.75 v 48 h 7.25 v -48 z"
       id="path4" />
  </g>
```
  
</details>

However, kvantum seems to ignore the second path completely, resulting in this:
![20240322_16h39m51s](https://github.com/tsujan/Kvantum/assets/43317083/35009e4d-b135-4907-8be8-9c8d87c8afa0)

What is the proper way to add such borders?

**Comment by tsujan** [comment]
> couldn't find anything in the making-themes pdf

Because your question isn't about Kvantum, but about SVG.

See the SVG file of `KvAmbience` for an example of a thin border around menu.
